src/mscorlib/shared/System/IO/StreamReader.cs

```diff
@@ -1127,7 +1127,7 @@ public override ValueTask<int> ReadAsync(Memory<char> buffer, CancellationToken
                         {
                             Debug.Assert(_bytePos == 0, "_bytePos can be non zero only when we are trying to _checkPreamble.  Are two threads using this StreamReader at the same time?");
 
-                            _byteLen = await tmpStream.ReadAsync(tmpByteBuffer, 0, tmpByteBuffer.Length, cancellationToken).ConfigureAwait(false);
+                            _byteLen = await tmpStream.ReadAsync(new Memory<byte>(tmpByteBuffer), cancellationToken).ConfigureAwait(false);
```

Comment:
nit: tmpByteBuffer.AsMemory() is cleaner (generic-type inference)

here and elsewhere

Reply:
> tmpByteBuffer.AsMemory() is cleaner (generic-type inference)

I prefer the explicitness.